.circleci/config.yml

```diff
@@ -609,33 +609,53 @@ jobs:
   e2e-tests:
     docker:
       - image: cimg/base:2021.03
+    parallelism: 3
```

Comment:
Using 3 containers (not more) because currently, there's still extra unneeded 5.5 minutes inefficiency (build, create_tenant) in every container.